specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,575 @@
         }
       }
     },
+    "AcsRouterJobCancelledEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobCancelled event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "note": {
+          "description": "Router Job Note",
+          "type": "string"
+        },
+        "dispositionCode": {
+          "description": "Router Job Disposition Code",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobClassificationFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassificationFailed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "errors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterJobClassifiedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassified event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "queueInfo": {
```

Comment:
Can this still be renamed? If so we should use queueDetails. If not we can add "x-ms-client-name" and set it to queueDetails.